app/views/pages/developer-guide/pos-product-marketplace/generators.liquid

```diff
@@ -0,0 +1,89 @@
+---
+converter: markdown
+metadata:
+  title: Generators
+  description: Using a generator to create boilerplate files for your project.   
+---
+
+To improve the workflow for creating a new resource you can use a generator that will create boilerplate files for your project.
+
+Generators follow development guidelines.
+
+
+### CRUD
+
+The CRUD generator will create all files needed to create, update, and delete.
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+./scaffold/bin/generate resource --help
+	Usage: generate <model_name> <attribute_name:type_attribute...> ex. generate car car_model:string year:integer
```

Comment:
@diana-lakatos 
Replacing < > with [ ] would make it visible in docs. Now its cut off because its "html-like" syntax

![image](https://user-images.githubusercontent.com/546845/106909697-d4a24f80-6700-11eb-8a85-800119e506d6.png)